api/v1beta1/disruption_webhook.go

```diff
@@ -170,3 +189,135 @@ func (r *Disruption) getMetricsTags() []string {
 
 	return tags
 }
+
+// initialSafetynets runs the initial safety nets for any new disruption
+// returns true if any safety net were caught and returns any errors when attempting to run the safety nets
+func (r *Disruption) initialSafetynets() ([]string, error) {
```

Comment:
```suggestion
func (r *Disruption) initialSafetyNets() ([]string, error) {
```

For consistency with other camelCasing of safetyNet